Home Page Content

```diff
@@ -0,0 +1,64 @@
+
+
+For Property Managers, By Property Managers …
+Welcome To The World Of Loca
+
+
+
+
+
+An Advanced Web-based, Open Source Management Application Created Specifically for Rental Property Owners, Landlords and Managers
+  
+Available for Windows, Mac and Linux Users.
```

Comment:
As it is a web-based app, I think it's useless to mention those Operating Systems.